USAGE.md

```diff
@@ -0,0 +1,561 @@
+layers to inherit properties from:
+
+- <b>Domain:</b> choose the domain and version for the new layer. Layers can only inherit properties from other layers
+  of the same domain and version.
+
+- <b>Score Expression:</b> Technique <a href="#scoring-techniques">scores</a> in the created
+  layer are initialized to the result of this expression. This field should take the form of an equation or constant,
+  using variables for layers.Layer variables are shown in yellow on tabs when the "create layer from other layers" panel
+  is opened:  
+  <img src="nav-app/src/assets/image_scoreVariableExample.png" height="100px">
+
+  Each technique's score is created independently using the score expression. For example, with a score expression
+  of <code>a+b</code>, some technique <i>t</i> in the output layer would be the sum of <i>t<sub>a</sub></i> and <i>
+  t<sub> b</sub></i>.
+
+  Expressions can also be comparative, for example <code>a>b</code> will create a layer with a score of 1
+  whereever <code><a>b</code> and 0 whereever <code>a<=b</code>. Such boolean expressions can be extended using <code>
+  and</code>, <code>or</code>, <code>xor</code> and <code>not</code>. You can also use ternary expressions such
+  as <code>a > b ? 25 : 10</code>. See <a href="http://mathjs.org/docs/expressions/syntax.html#operators">this page</a>
+  for a full list of operators.
+
+  Some example score expressions:
+    - <code>(a+b)/2</code> (average two layers)
+    - <code>a+b+c</code>
+    - <code>a*(b+c)</code>
+    - <code>100-a</code> (reverse the scoring of <code>a</code>, assuming <code>a</code> is on a 0-100 scale)
+    - <code>(a>b) and (a>=75)</code>
+
+- <b>Gradient:</b> choose which layer to inherit <a href="#scoring-gradient">scoring
+  gradient</a> from.
+- <b>Coloring:</b> choose which layer to inherit <a href="#assigning-manual-colors">manually
+  assigned technique colors</a>
+- <b>Comments:</b> choose which layer to inherit <a href="#adding-comments-to-techniques">
+  technique comments</a> from.
+- <b>States:</b> choose which layer to inherit technique <a href="#disabling-techniques">
+  enabled/disabled states</a> from.
+- <b>Filters:</b> choose which layer to inherit <a href="#filtering">layer filter
+  configuration</a> from.
+- <b>Legend:</b> choose which layer to inherit <a href="#legend">legend items</a> from.
+
+Tactic-spanning Techniques are evaluated individually: if a technique is annotated differently in two tactics, the
+output layers' techniques will honor this difference.
+
+<b>Tip:</b> Score expressions don't need to use variables! You can use this to create a new layer with a constant score
+for each technique. For example, if you wanted a new layer where all techniques are scored 50, you could simply type 50
+into the score expression input.
+
+# Layer Controls
+
+## ![Layer Information](nav-app/src/assets/icons/ic_description_black_24px.svg) Layer Information
```

Comment:
Only small issue left to fix here is that in GH this heading anchor gets generated as 'user-content--layer-information' (because of the icon image markdown), which causes all the anchor links in other places back to this header not work correctly

Reply:
This is still an observed issue as far as I can tell?